salt/state.py

```diff
@@ -3450,7 +3450,7 @@ def __contains__(self, saltenv):
     def __getitem__(self, saltenv):
         if saltenv != "base":
             self._fill()
-        if self._avail[saltenv] is None:
+        if saltenv not in self._avail or self._avail[saltenv] is None:
```

Comment:
Missed the review period - I was going to suggest this could be simplified as:
```suggestion
        if self._avail.get(saltenv) is None:
```